tests/functional/TestSmokeTest/vagrant/roles/common/tasks/main.yml

```diff
@@ -10,11 +10,16 @@
     - /dev/vdh
     - /dev/vdi
 
-- name: install packages
+- name: install repo source packages
   yum: name={{ item }} state=present
   with_items:
     - epel-release
-    - centos-release-gluster
+    - centos-release-storage-common
+
+- name: install repo source packages (phase 2)
+  yum: name={{ item }} state=present
+  with_items:
+    - centos-release-gluster5
```

Comment:
I manually retriggered the last run through the Jenkins webui. It worked now. I expect that your 1st patch also succeeds now the (internal) mirroring has been corrected.